app/src/main/java/com/x8bit/bitwarden/ui/vault/feature/itemlisting/VaultItemListingViewModel.kt

```diff
@@ -314,6 +328,15 @@ class VaultItemListingViewModel @Inject constructor(
                     cipherView = cipherView,
                 )
             }
+            ?: specialCircumstanceManager
```

Comment:
We should rename this method to account for the fact that it can either register or assert a credential